develop.md

```diff
@@ -1,49 +1,195 @@
-Develop
-=======
+Guidelines and documentation for developers of the SonarQube Docker images.
 
-Helpful notes for developers of the images.
 
-Releasing a new version of an image
------------------------------------
+Adding images for a new version of SQ
+=====================================
 
-Create the new image for the new version:
+New major version
+-----------------
 
-1. Rename the last version's directory, for example `7.8-community` to `7.9-community`
-2. Update the version in `Dockerfile`, and make any other necessary changes (usually none)
-3. Update the version in `.travis.yml`
-4. Run tests, for example: `./run-tests.sh 7.9-community`
-5. Create a Pull Request, confirm Travis build passes, get it reviewed and merged
+Create a new subdirectory with the major version number in the root directory of the repository with a sub directory for each supported edition, eg.: 
 
-Update the public repositories for [Docker Hub][hub]:
+```
+mkdir -p 9/community 9/developer 9/enterprise
+```
 
-1. Only if necessary, update files in https://github.com/docker-library/docs/tree/master/sonarqube and create a Pull Request
-    1. The markdown format must follow a certain standard, otherwise automated tests will fail. You can test with the `markdownfmt.sh` tool included in the repository, for example `./markdownfmt.sh -d sonarqube/content.md` will output the diff that would have to be done to make the tests pass. You can use the `patch` command to apply the changes, for example: `./markdownfmt.sh -d sonarqube/content.md | patch sonarqube/content.md`
-    2. Verify the Pull Request passes the automated tests (visible in the status of the PR)
+Each edition directory will contain a single DockerFile.
 
-2. Update https://github.com/docker-library/official-images/blob/master/library/sonarqube
-    1. Update the `Tags` and `Directory` entries appropriately
-    2. Update the commit sha to the just merged commit in `docker-sonarqube`
-    3. Create a Pull Request, with a simple summary of the new release
-    4. Verify the Pull Request passes the automated tests (visible in the status of the PR)
+New non-major version
+---------------------
 
-3. Wait for the Pull Requests to get merged, and verify the updated page on [Docker Hub][hub]
+As of today, we publish only a single version of SQ for a given major version in the "master" branch.
 
-FAQ
----
+New new image is therefor created for non-major versions of SQ. Instead, existing images are updated.
```

Comment:
```suggestion
New new image is therefore created for non-major versions of SQ. Instead, existing images are updated.
```

Reply:
`Instead`?

Reply:
"New new" was intended to be "No new"

Indeed, with that typo, both sentences hardly make any sense